bad_code_test.py

```
@@ -0,0 +1,60 @@
+    if radius < 0:
+        print("Invalid radius!")  # Violates again
+        return None
+    area = math.pi * radius * radius
+    for i in range(60):  # Just to exceed line length rule
+        area += 0  # Doing nothing useful
+    return area
+
+
+# Another function with no docstring
+def logicIssue():
+    result = eval("2+2")  # Violates disallow_eval_usage
```

Ruff (0.14.4)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)